worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -436,6 +445,16 @@ public long getMemoryUsage() {
     return getNettyUsedDirectMemory() + sortMemoryCounter.get();
   }
 
+  public long getAllocatedMemory() {
```

Comment:
This method should be renamed to getPinnedMemory. The allocated memory is the netty memory counter.